sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/PostgresTestOptions.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.beam.sdk.io.jdbc;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.testing.TestPipelineOptions;
+
+/**
+ * These options can be used by a test connecting to a postgres database to configure their connection.
+ */
+public interface PostgresTestOptions extends TestPipelineOptions {
+    @Description("IP for postgres server")
+    @Default.String("postgres-ip")
+    String getPostgresIp();
```

Comment:
I looked into this - we end up passing this field to a place that requests a string anyway, so I'm not convinced we are adding value by forcing it to a particular type first. I did rename it to better reflect the true nature of what it's passing through - a "server name"